app/models/hyrax/file_set.rb

```diff
@@ -45,6 +45,13 @@ def self.model_name(name_class: Hyrax::Name)
     attribute :original_file_id, Valkyrie::Types::ID.optional # id for FileMetadata resource
     attribute :extracted_text_id, Valkyrie::Types::ID.optional # id for FileMetadata resource
 
+    attribute :admin_set_id,             Valkyrie::Types::ID
+    attribute :member_ids,               Valkyrie::Types::Array.of(Valkyrie::Types::ID).meta(ordered: true)
+    attribute :member_of_collection_ids, Valkyrie::Types::Set.of(Valkyrie::Types::ID)
+    attribute :on_behalf_of,             Valkyrie::Types::String
+    attribute :proxy_depositor,          Valkyrie::Types::String
+    attribute :rendering_ids,            Valkyrie::Types::Array.of(Valkyrie::Types::ID).meta(ordered: true)
+
```

Comment:
All of these attributes are currently required by `Hyrax::ResourceForm`, but they aren’t present in `Hyrax::FileSet` currently. Should they be moved off of `Hyrax::ResourceForm` somehow, or do `Hyrax::FileSet`s need more work to support these?